python/glow/wgr/functions.py

```diff
@@ -157,7 +158,8 @@ def pivot_for_gwas(spark: SparkSession, label_df: pd.DataFrame) -> DataFrame:
         transposed_df = label_df.T
```

Comment:
Does this guarantee ordering?

Reply:
Yes, transposing does not sort the columns or rows.